ApplyToBecomeInternal/ApplyToBecomeInternal/Pages/TaskList/Decision/WhoDecided.cshtml.cs

```diff
@@ -26,8 +27,15 @@ public class WhoDecidedModel : DecisionBaseModel
 		[BindProperty, Required(ErrorMessage = "Please select who made the decision")]
 		public DecisionMadeBy? DecisionMadeBy { get; set; }
 
-		public IEnumerable<DecisionMadeBy> DecisionMadeByOptions => Enum.GetValues(typeof(DecisionMadeBy))
-																	.Cast<DecisionMadeBy>();
+		public IEnumerable<DecisionMadeBy> DecisionMadeByOptions => new List<DecisionMadeBy>
```

Comment:
And then you don't need to have the nested references:
`
public IEnumerable<DecisionMadeBy> DecisionMadeByOptions => new List<DecisionMadeBy>
		{
			// Reorder the way the radio buttons are displayed
			RegionalDirectorForRegion,
			OtherRegionalDirector,
			DirectorGeneral,
			Minister, 
			None,
		};`